### DISCUSSION TITLE
Don't render normal markdown when using LaTeX expressions

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#16939] [Other Feature Feedback, Questions, & Ideas]
If possible should GitHub not render any normal markdown formatting character when it is used inside the `$` symbol.

The reason is simple: When using LaTeX expressions will you probably use `*` for multiplication. Using this character in the text field will however show every text after it in an *italic tilt* even tho it isn't used to format text.

While this visual behaviour only seems to be the case with editing files (Discussion, issue and PR text fields seem to not render markdown at all) and only is a visual glitch for the "Write" box, is it still annoying and should - if possible - be fixed.

Here's a screenshot showcasing this behaviour:
![image](https://user-images.githubusercontent.com/11576465/169400338-067719ff-4c13-4977-8a9c-cb4d996303bf.png)

**Comment by UlyssesZh** [comment]
I think that will be difficult to change the order of processing because MathJax always kicks in last (after the Markdown finishes rendering). I think the only way of fixing that order issue is to ask GitHub to rewrite the rendering logic of Markdown, and that would lead to a brand-new flavor of Markdown. We should not expect this to happen (at least in the near future).
Currently, there is a workaround. You can use backslash to escape the symbols `*`, `_`, etc.

**Reply by Andre601** [reply]
> I think that will be difficult to change the order of processing because MathJax always kicks in last (after the Markdown finishes rendering). I think the only way of fixing that order issue is to ask GitHub to rewrite the rendering logic of Markdown, and that would lead to a brand-new flavor of Markdown. We should not expect this to happen (at least in the near future).
> Currently, there is a workaround. You can use backslash to escape the symbols `*`, `_`, etc.

I doubt that.
My proposal is to change the live-rendering that is only active in md files snyways, so it could just be disabled.
And I really doubt that the logic would be *that* hard here.

**Reply by avm19** [reply]
> I think that will be difficult to change the order of processing because MathJax always kicks in last (after the Markdown finishes rendering). I think the only way of fixing that order issue is to ask GitHub to rewrite the rendering logic of Markdown, and that would lead to a brand-new flavor of Markdown. We should not expect this to happen (at least in the near future). Currently, there is a workaround. You can use backslash to escape the symbols `*`, `_`, etc.

I don't think stackexchange.com / stackoverflow.com has any problems with this.

**Comment by avm19** [comment]
This issue (or a very similar one) is known to the support. I included their response in the sister thread https://github.com/github-community/community/discussions/18897#discussion-4156463
